server/routes/shared_api/observation/series.py

```diff
@@ -42,26 +43,36 @@
 bp = Blueprint("series", __name__, url_prefix='/api/observations/series')
 
 
-@bp.route('', strict_slashes=False)
+# Filters a list for non empty values
+def _get_filtered_arg_list(arg_list: List[str]) -> List[str]:
+  return list(filter(lambda x: x != "", arg_list))
```

Comment:
maybe for future PR: you can use `request` in this function directly and this function can just pass in the arg name?

Reply:
added todo

Reply:
+1.  also use constants for field-names when we do that